### ISSUE TITLE
Filter on auto-invite

### ISSUE THREAD

**Issue by sinkda** [DomenikIrrgang/ClassicLFG#64]
With the auto-invite feature people seem to have a lot of issues with it being able to form groups without interacting with people.   I see that issue #60 is meant to deal with that by making it work on custom only.   is it possible to add 2 additional fields for auto-invite in the options that say auto-invite ONLY if:

1.) person is in your guild
2.) person is on your friend list

Or have an in-addon list of "whitelist" people who can auto-join a group.  This way people can further fine tune the auto invite.

Keep up the great work on this addon, despite what some people may think of it :)

**Comment by DomenikIrrgang** [comment]
I like the idea for friends and guild mates only, I think the whitelist is a bit to unhandy! Thanks for the support and suggestion!

**Comment by sinkda** [comment]
I agree as it would cause people to have to keep a "friends" list and a "whitelist" list for players though in theory the whitelist could be longer than the friends list.  Was just throwing it out there but I think a guild / friend list only option would be a solid foundation start that might make people more open to the idea.

**Comment by DomenikIrrgang** [comment]
I removed the auto invite completly for now, will probably bring it back with the limitations stated here if the new API changes are going to allow for it.
